sdk/spring/azure-spring-data-cosmos/pom.xml

```diff
@@ -85,10 +85,21 @@
       <artifactId>spring-expression</artifactId>
       <version>6.0.0-M5</version> <!-- {x-version-update;org.springframework:spring-expression;external_dependency} -->
     </dependency>
+    <dependency>
+      <groupId>io.micrometer</groupId>
```

Comment:
why do we need micrometer in spring-data-cosmos?

Reply:
It's used to fix the issue  https://github.com/micrometer-metrics/micrometer/issues/3398

Reply:
But shouldn't it be sufficient to add this as "provided" vs. making it a transitive dependency?